concert/processes/common.py

```diff
@@ -216,20 +216,20 @@ async def acquire_frames_360(camera, rotation_motor, num_frames, shutter=None, f
          position_eps=Numeric(1, q.deg), max_iterations=Numeric(1),
          initial_x_coeff=Numeric(1, q.dimensionless), initial_z_coeff=Numeric(1, q.dimensionless),
          shutter=Shutter, flat_motor=LinearMotor, flat_position=Numeric(1, q.m),
-         y_0=Numeric(1), y_1=Numeric(1), get_ellipse_points_kwargs=None, frame_consumer=None)
+         y_0=Numeric(1), y_1=Numeric(1), get_ellipse_points_kwargs=None, frame_consumers=None)
```

Comment:
Just for curiosity: What was your use case here for multiple frame_consumers? Why not just using a broadcast coro?

Reply:
I was storing in an accumulator and sending to live view at the same time. Then I showed `viewer.show(np.mean(acc.items[10:20]))` to show how the needle travelled (360 deg in 10 projections).